src/api/undeploy/controllers/index.js

```diff
@@ -0,0 +1,3 @@
+import { undeployServiceController } from '~/src/api/undeploy/controllers/undeploy-service.js'
+
+export { undeployServiceController }
```

Comment:
Is this needed?

Reply:
Meh, Can be explicit in the router to the full controller path however once we add another controller we have to change it all again...

Reply:
I've been really trying to cut down on excessive `index.js` files. Same same, i'd say this isn't needed